src/main/java/com/griddynamics/cloud/learning/dal/repositories/ArticleRepository.java

```diff
@@ -0,0 +1,121 @@
+package com.griddynamics.cloud.learning.dal.repositories;
```

Comment:
These queries don't fit our business requirements, let's review the cases together